include/deal.II/base/synchronous_iterator.h

```diff
@@ -38,6 +41,12 @@ DEAL_II_NAMESPACE_OPEN
  * The template argument of the current class shall be of type
  * <code>std_cxx11::tuple</code> with arguments equal to the iterator types.
  *
+ * The individual iterators can be accessed using 
+ * <code>std_cxx1::get<X>(synchronous_iterator.iterators)</code> where X is the 
+ * number corresponding to the desired iterator.
+ *
+ * Additional documentation can be found under Files then File Members.
```

Comment:
I think this is too complicated. Are you saying that you'd like to group global functions with the class declared here? Could you use `@relates` instead?